test/CollapseSpec.js

```diff
@@ -184,7 +182,10 @@ describe('<Collapse>', () => {
 
       wrapper.setState({ dimension });
 
-      assert.equal(wrapper.instance().collapse.getDimension(), 'whatevs');
+      assert.equal(
+        collapseHelpers.getDimension(wrapper.find(Collapse).props().dimension),
+        'whatevs',
+      );
```

Comment:
**suggestion:** per https://github.com/moh12594/react-bootstrap/blob/2329c05768f09104c3c6453b875abb61902c9db8/src/Collapse.js#L236, probably better to just set the dimension to `width`, and assert that the `width` class is present